Makefile

```diff
@@ -383,8 +379,7 @@ build-controller-ubi:
 .PHONY: build-windows
 build-windows:
 	@echo "===> Building Antrea bins and antrea/antrea-windows Docker image <==="
-	docker build -t antrea/antrea-windows:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.build.windows --network $(DOCKER_NETWORK) $(WIN_BUILD_ARGS) .
-	docker tag antrea/antrea-windows:$(DOCKER_IMG_VERSION) antrea/antrea-windows
+	build/images/base-windows/build.sh --target agent --agent-tag $(DOCKER_IMG_VERSION) $(WINDOWS_BUILD_ARGS)
```

Comment:
I am not sure if this is a proper way to call build script in Makefile, should we run `make` command in build script and `docker build` command in Makefile?

Reply:
What's confusing is that we are using the build script for the base image (it's under `base-windows/`) to build the Windows image for the Antrea Agent?

Reply:
> I am not sure if this is a proper way to call build script in Makefile, should we run make command in build script and docker build command in Makefile?

I called the script in makefile directly, because we shall create or switch a buildx instance dedicated for Windows, and better to switch to previous buildx instance after Windows build completed. I don't feel it is good to introduce complicated logic in a `make` command. For existing commands defined in Makefile, there exist other commands to call script, e.g., make codegen, make manifest.

> What's confusing is that we are using the build script for the base image to build the Windows image for the Antrea Agent?

The fact is the build logic for base image and Agent image is similar, and the differences mainly exist in the dockerfile, image name, and parameters. So I move the logic together. If the confusion is the path of script is in base-windows, is it better to move it to path `build/images/build-windows.sh` and then we use it for both base and agent images?

Reply:
> What's confusing is that we are using the build script for the base image to build the Windows image for the Antrea Agent?

The fact is the build logic for base image and Agent image is similar, and the differences mainly exist in the dockerfile, image name, and parameters. So I move the logic together. If the confusion is the path of script is in base-windows, is it better to move it to path `build/images/build-windows.sh` and then we use it for both base and agent images? @antoninbas

Reply:
Using build/images/base-windows/build.sh directly inside target should be acceptable , however this file would need executable permission !, since script is common one to build base and agent image, it could be preferred to keep it in a common location like build/images